frontend/app/student/run-simulation/[instanceId]/page.tsx

```
@@ -1013,95 +1429,242 @@ ${availablePersonas.map(persona => `• @${persona.name.toLowerCase().replace(/\
+        <RoleBasedSidebar currentPath={`/student/run-simulation/${instanceId}`} />
+        
+      <div className="flex-1 ml-20 flex flex-col">
+        {/* Top Navigation Bar */}
+        <div className="bg-white px-6 py-4">
+          <div className="flex items-center justify-between">
+            <div className="flex items-center gap-4">
+              <button
+                onClick={() => router.push("/student/simulations")}
+                className="text-gray-600 hover:text-gray-900"
+              >
+                <ArrowLeft className="w-5 h-5" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)